settings.gradle.kts

```diff
@@ -15,34 +15,45 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+import com.gradle.enterprise.gradleplugin.internal.extension.BuildScanExtensionWithHiddenFeatures
 
 plugins {
-  id("com.gradle.enterprise") version "3.4.1" apply false
+  id("com.gradle.enterprise") version "3.13.2"
+  id("com.gradle.common-custom-user-data-gradle-plugin") version "1.10"
 }
 
 
-// Plugins which require online access should not be enabled when running in offline mode.
-if (!gradle.startParameter.isOffline) {
```

Comment:
Is there a reason we don't need this anymore?

Reply:
The build scam publishing is implicitly disabled when offline is passed. So the check is not needed and omitting it can simplify your build scripts